website/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/exome.methods.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+---
+
+# Exome Germline Single Sample v2.4.4 Methods
+
+The following contains a detailed methods description outlining the pipeline’s process, software, and tools that can be modified for a publication methods section.
+
+## Detailed Methods
+
+Preprocessing and variant calling was performed using the ExomeGermlineSingleSample 2.4.4 pipeline using Picard 2.23.8, GATK 4.1.8, and Samtools 1.11 with default tool parameters unless otherwise specified. All reference files are available in the public [Broad References Google Bucket](https://console.cloud.google.com/storage/browser/gcp-public-data--broad-references/hg38/v0). The pipeline follows GATK Best Practices as previously described ([Van der Auwera & O'Connor, 2020](https://www.oreilly.com/library/view/genomics-in-the/9781491975183/)) as well as the Functional Equivalence specification ([Regier et al., 2018](https://www.nature.com/articles/s41467-018-06159-4)).
+
+### Pre-processing and QC
+
+Prior to pre-processing and quality control, contamination resources were subsetted according to interval list for the exome preparation using BedTools 2.27.1. Exome paired-end reads in unmapped BAM (uBAM) format were first scattered to perform QC and alignment in parallel. Quality metrics were calculated using Picard CollectQualityYieldMetrics. uBAMs were converted to FASTQ using Picard SamToFastq and aligned to the Hg38 reference genome using BWA mem 0.7.15 with batch size set using -K 100000000. Metadata from the uBAMs was then merged with the aligned BAMs using Picard MergeBamAlignment with the parameters --SORT_ORDER="unsorted", allowing the data to be grouped by read name for efficient downstream marking of duplicates, and --UNMAP_CONTAMINANT_READS=true, to remove cross-species contamination.
+
+QC metrics (base distribution by cycle, insert size metrics, mean quality by cycle, and quality score distribution) were collected for the aligned, unsorted read-groups using Picard CollectMultipleMetrics. The read-group specific aligned BAMs were then aggregated and duplicate reads were flagged using Picard MarkDuplicates assuming queryname-sorted order and the parameter --OPTICAL_DUPLICATE_PIXEL_DISTANCE=2500, which is appropriate for patterned flowcells.
+
+The aggregated BAM file was then sorted using Picard SortSam with coordinate sort order. The fingerprints of separate read groups were verified using Picard CrosscheckFingerprints with a LOD threshold of -10. Cross-sample contamination was checked using verifyBamID2.
```

Comment:
Do we want to link to the paper on fingerprinting here? https://www.nature.com/articles/s41467-020-17453-5

Reply:
Added this Exome and WGS. Thank you!